docs/README.md

```diff
@@ -60,13 +60,18 @@ Kevin Zolman
 
 
 # 💭 Abstract
-<!-- TBD -->
+This application is a Dungeons and Dragons Encounter Builder, with the intended user being a Dungeon Master. It allows the user to fill out a form to add encounter details including encounter name, party size, party level, summary, description, treasure & rewards, and monsters. Monsters from the D&D 5e compendium can be filtered and added based on size, name, hit points, and armor class attributes. 
 
```

Comment:
Might want to also add a line about the user being able to preview encounters that they've built, and then click to see full details of each encounter.